src/frontend/src/components/Posts/Posts.jsx

```diff
@@ -15,13 +15,8 @@ const useStyles = makeStyles((theme) => ({
   },
   content: {
     '& > *': {
-      /**
-       * We, the implementors of this CSS realize how morally wrong it is
-       * to use !important in any case. That does not excuse the hour long
-       * Fight while finding other ways
-       */
-      color: `${theme.palette.secondary.light} !important`,
-      borderColor: `${theme.palette.secondary.light} !important`,
+      color: 'primary',
+      borderColor: 'primary',
```

Comment:
What does the string `'primary'` mean vs. using the property `theme.palette.text.primary`?

Reply:
For that one, I made the changes before I knew I could use the theme.palette property. The 'primary' works in certain situations but I'm not sure which ones, so I'll change this for better consistency

Reply:
I'm wondering if `primary` is actually the default Material UI blue

Reply:
primary is whatever you set it up, I just don't know when you can just use 'primary' and when you need theme.palette